libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -130,7 +129,8 @@ typedef struct IotI2CDescriptor   * IotI2CHandle_t;
  *                          passed back to the caller in the callback.
  *
  */
-typedef void (* IotI2CCallback_t) ( IotI2COperationStatus_t xOpStatus, void * pvUserContext );
+typedef void (* IotI2CCallback_t) ( IotI2COperationStatus_t xOpStatus,
+                                    void * pvUserContext );
 
 /**
```

Comment:
I feel like we should add a comment here that the expectation is that there will be one instance for each I2C device.  It is technically possible that someone could have one instance to support multiple devices on the same bus, as long as they changed/set the slave address between each operation.  But, this would require a higher level to synchronize the operations.